pygeos/geos.py

```diff
@@ -0,0 +1,26 @@
+from . import lib
+from functools import wraps
+
+
+class UnsupportedGEOSOperation(ImportError):
+    pass
+
+
+class requires_geos:
+    def __init__(self, version):
+        if version.count(".") == 1:
```

Comment:
@caspervdw since this annotation is used by pygeos contributors (rather than users), it seems reasonable to insist on standard usage (always <major>.<minor>.<patch>) instead of inferring patch component, e.g., 
```
if version.count(".") != 2:
    raise ValueError("version must be <major>.<minor>.<patch> format")
```

otherwise as written, this doesn't handle missing minor component...

Also seems like it would be good to parse version into a tuple here for internal use, so that `TypeError`s are caught in the annotation constructor (e.g., if can't be coerced to int) rather than in `__call__`